azure-pipelines.yml

```
@@ -54,13 +56,40 @@
+#              path: $(dependencyCheckDataDir)
+#              cacheHitVar: CACHE_HIT
+#          - script: |
+#              if [ '$(CACHE_HIT)' = 'true' ]; then
+#                echo "Restored cache."
+#              else
+#                echo "No cache found"
+#              fi
+#            displayName: 'Log cache hit'
           - script: |
               echo ">> ALL BUILD* VARIABLES " && export | grep BUILD
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)